### PR TITLE
Fix Add New Show redirection

### PR DESCRIPTION
**Pull request by p0psicles**
- [ ] PR is based on the DEVELOP branch
- [ ] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [ ] Read the [contribution guide](https://github.com/pymedusa/Medusa/blob/master/.github/CONTRIBUTING.md)

### CODE REVIEW

File: themes-default/slim/static/js/vue-submit-form.js [hunk 1]
```diff
@@ -13,8 +13,13 @@ window.vueSubmitForm = async function(formId) {
     // [pair[0]]: pair[1]
     // }), {});
     this.$http[method](path, { body: formData, redirect: 'follow' })
-        .then(() => {
-            if (redirect) {
+        .then(res => {
+            // We can get the redirect url using obj.url.
+            if (res.url) {
+                window.location.href = res.url;
```
[the remark this answers is not in the document]
Comment (p0psicles) [reply themes-default/slim/static/js/vue-submit-form.js]: At this point i'm kinda lost. @OmgImAlexis mentioned that we should move away from any server side redirect, as that won't work with 100% vue. 

But I don't want to change 10 things at the same time. So i'd like to have something working as for the transitioning.

Frisbee should also be able to recognize a redirect in the headers right? Also if it's a post?

**Comment by sharkykh** [comment]
@p0psicles BTW I did check, and `self.redirect()` basically returns a `Location: url` header + a status code of either 301 or 302, which is redirection.
Just in case you needed that info.

File: themes-default/slim/static/js/vue-submit-form.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (sharkykh) [reply themes-default/slim/static/js/vue-submit-form.js]: >But I don't want to change 10 things at the same time. So i'd like to have something working as for the transitioning.

I agree.

>Frisbee should also be able to recognize a redirect in the headers right? Also if it's a post?

Should is the key word.
And, I was wrong - there's only one property: `redirected`, which is Boolean.
Don't know where I found that information, but [this is the correct one](https://developer.mozilla.org/en-US/docs/Web/API/Response/redirected), I believe.

**Comment by p0psicles** [comment]
Haha. Creative

**Pull request merged**